From 5e0b03bdb4c37d2eb35c999a96bbe829eb3e6a28 Mon Sep 17 00:00:00 2001
From: Carlos Cordoba <ccordoba12@gmail.com>
Date: Wed, 24 Jun 2015 16:01:14 -0500
Subject: [PATCH] Fix errors in PyGMO

---
 PyGMO/algorithm/__init__.py | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/PyGMO/algorithm/__init__.py b/PyGMO/algorithm/__init__.py
index 4a50d35..5895a2e 100644
--- a/PyGMO/algorithm/__init__.py
+++ b/PyGMO/algorithm/__init__.py
@@ -1046,9 +1046,9 @@ def _cstrs_co_evolution_ctor(
     """
     arg_list = []
     if original_algo is None:
-        original_algo = algorithm.jde()
+        original_algo = _algorithm.jde()
     if original_algo_penalties is None:
-        original_algo_penalties = algorithm.jde()
+        original_algo_penalties = _algorithm.jde()
     arg_list.append(original_algo)
     arg_list.append(original_algo_penalties)
     arg_list.append(pop_penalties_size)
@@ -1102,9 +1102,9 @@ def _cstrs_immune_system_ctor(
     arg_list = []
 
     if algorithm is None:
-        algorithm = algorithm.jde()
+        algorithm = _algorithm.jde()
     if algorithm_immune is None:
-        algorithm_immune = algorithm.jde()
+        algorithm_immune = _algorithm.jde()
     arg_list.append(algorithm)
     arg_list.append(algorithm_immune)
     arg_list.append(gen)
@@ -1145,9 +1145,9 @@ def _cstrs_core_ctor(
     """
     arg_list = []
     if algorithm is None:
-        algorithm = algorithm.jde()
+        algorithm = _algorithm.jde()
     if repair_algorithm is None:
-        repair_algorithm = algorithm.jde()
+        repair_algorithm = _algorithm.jde()
     arg_list.append(algorithm)
     arg_list.append(repair_algorithm)
     arg_list.append(gen)
